hipify-clang/packaging/hipify-clang.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(hipify-clang)
+
+install(PROGRAMS @HIPIFY_INSTALL_PATH@/hipify-clang DESTINATION bin)
+install(DIRECTORY @HIPIFY_INSTALL_PATH@/include DESTINATION bin)
+
+#############################
+# Packaging steps
+#############################
+set(CPACK_SET_DESTDIR TRUE)
+set(CPACK_INSTALL_PREFIX "/opt/rocm/hip")
+set(CPACK_PACKAGE_NAME "hipify-clang")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Hipify-clang:Clang-based tool to translate CUDA source code into portable HIP C++ automatically")
```

Comment:
hipify-clang: a clang-based tool to translate CUDA source code into portable HIP C++ automatically